.github/CONTRIBUTING.md

```diff
@@ -21,11 +33,15 @@ You should use unit-tests for SublimeText by using [UnitTesting](https://github.
 
 1. Install the UnitTesting plugin
 2. Comment out or rename your own `MarkdownTOC.sublime-settings` so individual settings are not used during testing
-3. [Run tests](https://github.com/randy3k/UnitTesting-example#running-tests) 
+3. [Run tests](https://github.com/randy3k/UnitTesting-example#running-tests)
 4. Send Pull Request when tests pass
 
 All contributions are welcome
 
+## Useful Tools
+
+- [Gyazo](https://gyazo.com/en): Tool for making screen captures into animation GIF. Use it for showing how the function works.
```

Comment:
Suggestion: "Tool for making animated GIFs from screen captures".

Reply:
thanks!